changelog/18.0/18.0.0/release_notes.md

```diff
@@ -0,0 +1,303 @@
+    - [VTCtld and VTOrc reparenting stats](#vtctld-and-vtorc-reparenting-stats)
+  - **[VTTablet](#vttablet)**
+    - [VTTablet: New ResetSequences RPC](#vttablet-new-rpc-reset-sequences)
+  - **[Docker](#docker)**
+    - [Debian: Bookworm added and made default](#debian-bookworm)
+    - [Debian: Buster removed](#debian-buster)
+  - **[Durability Policies](#durability-policies)**
+    - [New Durability Policies](#new-durability-policies)
+
+## <a id="major-changes"/>Major Changes
+
+### <a id="breaking-changes"/>Breaking Changes
+
+#### <a id="local-examples-etcd-v3"/>Local examples now use etcd v3 storage and API
+In previous releases the [local examples](https://github.com/vitessio/vitess/tree/main/examples/local) were
+explicitly using etcd v2 storage (`etcd --enable-v2=true`) and API (`ETCDCTL_API=2`) mode. We have now
+removed this legacy etcd usage and instead use the new (default) etcd v3 storage and API. Please see
+[PR #13791](https://github.com/vitessio/vitess/pull/13791) for additional info. If you are using the local
+examples in any sort of long-term non-testing capacity, then you will need to explicitly use the v2 storage
+and API mode or [migrate your existing data from v2 to v3](https://etcd.io/docs/v3.5/tutorials/how-to-migrate/).
+
+### <a id="new-flag"/>New command line flags and behavior
+
+#### <a id="new-flag-toggle-ers"/>VTOrc flag `--allow-emergency-reparent`
+
+VTOrc has a new flag `--allow-emergency-reparent` that allows the users to toggle the ability of VTOrc to run emergency
+reparent operations. The users that want VTOrc to fix the replication issues, but don't want it to run any reparents
+should start using this flag. By default, VTOrc will be able to run `EmergencyReparentShard`. The users must specify the
+flag to `false` to change the behaviour.
+
+#### <a id="new-flag-errant-gtid-convert"/>VTOrc flag `--change-tablets-with-errant-gtid-to-drained`
+
+VTOrc has a new flag `--change-tablets-with-errant-gtid-to-drained` that allows users to choose whether VTOrc should change the
+tablet type of tablets with errant GTIDs to `DRAINED`. By default, the flag is false.
+
+This feature allows users to configure VTOrc such that any tablet that encounters errant GTIDs is automatically taken out of the
+serving graph. These tablets can then be inspected for what the errant GTIDs are, and once fixed, they can rejoin the cluster.
+
+#### <a id="new-ers-subflag"/>ERS sub flag `--wait-for-all-tablets`
+
+Running `EmergencyReparentShard` from the vtctldclient has a new sub-flag `--wait-for-all-tablets` that makes `EmergencyReparentShard` wait 
+for a response from all the tablets. Originally `EmergencyReparentShard` was meant only to be run when a primary tablet is unreachable.
+We have realized now that there are cases when the replication is broken but all the tablets are reachable. In these cases, it is advisable to 
+call `EmergencyReparentShard` with `--wait-for-all-tablets` so that it doesn't ignore one of the tablets.
```

Comment:
```suggestion
Running `EmergencyReparentShard` from `vtctldclient` has a new sub-flag `--wait-for-all-tablets` that makes `EmergencyReparentShard` wait 
for a response from all the tablets. Originally `EmergencyReparentShard` was meant to run only when a primary tablet is unreachable.
We have realized now that there are cases when the replication is broken but all the tablets are reachable. In these cases, it is advisable to 
call `EmergencyReparentShard` with `--wait-for-all-tablets` so that it does not ignore one of the tablets.
```